clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -279,6 +294,13 @@ public void disconnected(ClientDescriptor clientDescriptor) {
 
   @Override
   public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEntityMessage message) {
+    if(this.offHeapResourceIdentifiers.isEmpty()) {
+      return responseFactory.failure(
+          new IllegalStateException("Server started without any offheap resources defined." +
```

Comment:
Used `IllegalStateException` here and not any other specific types that were recently added because I feel that in this case the server is really in an illegal state. But please do suggest if there is something better.